### ISSUE TITLE
Locked Icons

### ISSUE THREAD

**Issue by benbenlaw** [kirderf1/InventoryFree#15]
**Describe the bug**
its not really a bug but i think that the locked icon should always render behind other items.

**To Reproduce**
1. grab item
2. move over locked item slot
3. held item goes behind the locked item slot icon

**Expected behavior**
should probably render on top

**Screenshots**
![image](https://github.com/kirderf1/InventoryFree/assets/42780877/e1f15dee-bb8e-4c93-a8b0-f3d9d202db15)

**Versions (please complete the following information):**
 - Forge version:  forge-1.20.1-47.1.22]
 - Mod Version: InventoryFree-1.20.1-3.2.0

**Additional context**

**Comment by kirderf1** [comment]
Looks like I raised the texture offset from 200 to 500 when updating to 1.20.1. Probably to make sure that it draws above something, but I don't remember what. Anyways, setting it back to 200 fixed it, and it seemed to already be rendering fine when drawing the lock over an item, so I'm not sure why I raised it in the first place.
I'll set it back and add a comment so that I won't forget to check this if I decide to change that texture offset in the future.
